lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -12,6 +12,18 @@
 from django.contrib.auth.models import User
 
 
+def get_user_from_identifier(identifier):
+    """
+     This function takes the string identifier and fetch relevant user object from database
+    """
+    identifier = identifier.strip()
+    if '@' in identifier:
```

Comment:
This is prior art that has been refactored from an in-line bit of code into a method.  I don't think we should force @arbabnazar to fix this here, now as we'd like to have the ability to add multiple people to the whitelist yesterday.